### DISCUSSION TITLE
Ability to drop / hide sign in with email

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#5275] [Ideas]
Hi, I think it would be great if there was a way to hide/remove the bottom sign-in by email section when single sign-on is working. it makes it confusing for new users. 
![image](https://user-images.githubusercontent.com/80058238/162027458-25854a08-28e2-485a-9ad7-43bea5f2b0e7.png)

**Comment by melohagan** [comment]
Hi @KernOil 

Great idea, and an issue was raised fairly recently: https://github.com/Budibase/budibase/issues/4859

It is scheduled to be worked on, and hopefully this will be available in the next release.

**Reply by KernOil** [reply]
good stuff thank you for the link!
